### DISCUSSION TITLE
Download photos and videos of Amazon products

### DISCUSSION THREAD

**Discussion by simonchaconb** [aclap-dev/video-downloadhelper#327] [Help and Q&A]
### Your question

Can I download images too? It is being useful for me to download videos from Amazon but I would also like to download the images

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: es-ES
license: accepted
key (only 16 first characters): eec4fe347841455c
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\usuario",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
Hi, yes you should be able to download images.
But I tested on amazon and the download didn't work.
@paulrouget can you have a look?

<img width="521" alt="Screenshot 2024-03-26 at 08 38 26" src="https://github.com/aclap-dev/video-downloadhelper/assets/368367/770fcb4a-dec1-4273-b504-31b8c2310d64">


<img width="642" alt="Screenshot 2024-03-26 at 08 36 19" src="https://github.com/aclap-dev/video-downloadhelper/assets/368367/abac2fbb-5299-4711-b9e8-704c1e48388d">

**Reply by simonchaconb** [reply]
This is the error that shows me when I try to download an image from Amazon (I don't know if it influences that I am logged in with my user on Amazon): **Cannot read properties of undefined (reading 'container')**
TypeError: Cannot read properties of undefined (reading 'container')
    at RT (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:44:9196)
    at async ri (chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/background/main.js:44:13259)
![Captura-error-downloadhelper](https://github.com/aclap-dev/video-downloadhelper/assets/164894706/64a3e506-1dff-47ed-b9a0-170f479621e3)
